code/game/objects/structures/kitchen_spike.dm

```diff
@@ -33,10 +38,12 @@
 	var/obj/item/stack/rods/used_rods = attacking_item
 	if(used_rods.get_amount() >= MEATSPIKE_IRONROD_REQUIREMENT)
 		used_rods.use(MEATSPIKE_IRONROD_REQUIREMENT)
-		to_chat(user, span_notice("You add spikes to the frame."))
+		balloon_alert(user, "you add spikes to the frame.")
```

Comment:
```suggestion
		balloon_alert(user, "meatspike built.")
```
There's already visual indication of what you did so I don't think this is needed at all, actually.